src/Core/Content/MeasurementSystem/MeasurementUnitTypeEnum.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\MeasurementSystem;
+
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @internal
```

Comment:
THis enum and the `ProductMeasurementEnum` is internal, because externally you only need to use the `ConvertedUnit`?  Is that correct, or are there cases where external code would need to rely on this? E.g. when they write unit data to the DB? 🤔